developer_docs/how_to_create_a_plugin.asciidoc

```diff
@@ -1330,6 +1330,19 @@ class RandomStatus < HostStatus::Status
 end
 ----
 
+The status class _must_ implement the followig methods:
+
+ * `to_label`: this method will be called to determine the string that will be used while displaying the status value.
+ * `self.status_name`: this method will be called to determine what label to display for the status.
+
+It _can_ also implement the following methods according to the specific needs:
+
+ * `to_global`: This method will be used to determine global status according to this specific one. The mechanism here is "voting" - to_global is called for each status and the bigger value from https://github.com/theforeman/foreman/blob/ceb276fbe96b97770f5d292b1eadca0205a34a0a/app/models/host_status/global.rb#L3[the list] would be taken. The default is `HostStatus::Global::OK`.
```

Comment:
Instead of `bigger value`, souldn't it be `highest value`?

Reply:
Right!